src/views/components/trace/trace-detail-chart-tree.vue

```diff
@@ -263,4 +290,8 @@ export default {
      fill: rgba(0,0,0,0.05)
    }
  }
+.dialog-c-text {
```

Comment:
Since the contents are exceptions, it would be more comfortable if the font is monospaced.

Reply:
It has been modified